sdcm/nemesis.py

```diff
@@ -1645,12 +1645,6 @@ def execute_disrupt_method(self, disrupt_method):
         self.metrics_srv.event_start(disrupt_method_name)
         try:
             disrupt_method()
-        except Exception as exc:  # pylint: disable=broad-except
```

Comment:
how about just adding:
```
except UnsupportedNemesis:
    raise
except Exception as exc:
...
```
so we don't change current logic and keep `UnsupportedNemesis` more quiet.

Reply:
there's no logic here, `error_list` isn't really being used (and the same error already gets appended else where)
and the debug print is just adding more confusion, i.e. same error printed again and again.

Reply:
on only reason for adding `except UnsupportedNemesis:` is for documentation, so people won't remove it